bench/softposit-rs/optimized/8mae6k72v4zmmji.ll

```diff
@@ -942,9 +942,10 @@ _ZN9softposit5p32e27convert22convert_u32_to_p32bits17h615e043045e072d1E.exit: ;
 20:                                               ; preds = %_ZN9softposit5p32e27convert22convert_u32_to_p32bits17h615e043045e072d1E.exit
   %21 = sub i32 0, %1
   %.057 = tail call i32 @llvm.abs.i32(i32 %1, i1 false)
-  %.not = icmp slt i32 %.057, 2126512128
-  %.not.i87 = icmp sgt i32 %1, -1
-  br i1 %.not, label %119, label %71
+  %.054 = tail call i8 @llvm.scmp.i8.i32(i32 %.057, i32 2126512128)
+  %switch = icmp ult i8 %.054, 2
```

Comment:
Duplicate of https://github.com/dtcxzyw/llvm-opt-benchmark/pull/1194#pullrequestreview-2238695484